quilt/tools/command.py

```diff
@@ -489,9 +489,9 @@ def _print_children(children, prefix, path):
     def _print_node(node, prefix, child_prefix, name, path):
         name_prefix = u"─ "
         if isinstance(node, dict):
-            node_type = NodeType(node.pop(TYPE_KEY))
+            node_type = NodeType(node["type"])
```

Comment:
Constants are kind of a workaround: the real solution is to use classes, and only use JSON for serialization/deserialization - but that's a bigger change.